common/mac/build.sh

```diff
@@ -23,6 +18,11 @@ builder_describe "Keyman common mac modules" \
 
 builder_parse "$@"
 
+if [[ $BUILDER_OS != mac ]]; then
+  builder_echo grey "Platform is not macOS; skipping common/mac"
+  exit 0
+fi
```

Comment:
A builder script must always call `builder_describe` and `builder_parse`, so do our platform checks after that. This allows the script to be called with `--help` even when on the wrong platform, so user can get info about it.